libs/math/tests/unit/basic_math/standard_functions_test.cpp

```diff
@@ -76,26 +76,27 @@ TYPED_TEST(StandardFunctionTests, clamp_array_1D_test)
 
   ArrayType A = ArrayType({6});
 
-  A(0) = DataType(-10);
-  A(1) = static_cast<DataType>(0);
-  A(2) = static_cast<DataType>(1);
-  A(3) = DataType(2);
-  A(4) = DataType(3);
-  A(5) = DataType(10);
+  A(0) = DataType{-10};
+  A(1) = DataType{0};
+  A(2) = DataType{1};
+  A(3) = DataType{2};
+  A(4) = DataType{3};
+  A(5) = DataType{10};
 
   // Expected results
   ArrayType A_clamp_expected = ArrayType({6});
-  A_clamp_expected(0)        = DataType(2);
-  A_clamp_expected(1)        = DataType(2);
-  A_clamp_expected(2)        = DataType(2);
-  A_clamp_expected(3)        = DataType(2);
-  A_clamp_expected(4)        = DataType(3);
-  A_clamp_expected(5)        = DataType(3);
+  A_clamp_expected(0)        = DataType{2};
+  A_clamp_expected(1)        = DataType{2};
+  A_clamp_expected(2)        = DataType{2};
+  A_clamp_expected(3)        = DataType{2};
+  A_clamp_expected(4)        = DataType{3};
+  A_clamp_expected(5)        = DataType{3};
 
   // Compare results with expected results
-  Clamp(DataType(2), DataType(3), A);
+  Clamp(DataType{2}, DataType{3}, A);
 
-  ASSERT_TRUE(A.AllClose(A_clamp_expected, DataType{1e-5f}, DataType{1e-5f}));
+  ASSERT_TRUE(A.AllClose(A_clamp_expected, fetch::math::Type<DataType>("0.00001"),
```

Comment:
I think there shouldn't be any tolerance in this test. It should just be ExpectEq(A, A_clamp_expected);